test/functional/fixtures/es_archiver/data/mappings.json

```diff
@@ -0,0 +1,450 @@
+{
```

Comment:
since you're using 

`test/functional/fixtures/es_archiver/unmapped_fields`

for your functionals, you can delete 

`test/functional/fixtures/es_archiver/data`